simulator/build.cmd

```diff
@@ -10,6 +10,7 @@ rem Build object files
 cl /I. /EHsc /c /nologo /MD ^
     /D__LIBELF_INTERNAL__=1 ^
    /W4 /WX /wd4505 /wd4244 /wd4996 /wd4267 ^
+    /std:c++17 ^
```

Comment:
There is one more place to add this flag, check main.cpp build.
Please remove one space in the beginning of the line.